ansible/roles/test/tasks/vnet_vxlan.yml

```diff
@@ -140,3 +140,42 @@
         - vxlan_enabled=True
         - config_file='/tmp/vnet.json'
         ptf_extra_options: "--relax --debug info --log-file /tmp/vnet_test.log"
+
+    - set_fact: op="DEL"
+
+    - name: Render DUT VNet route configuration
+      template: src=vnet_routes.j2 dest=/tmp/vnet.route.json
+
+    - block:
+        - name: Copy configuration to swss container
+          shell: docker cp /tmp/vnet.route.json swss:/vnet.route.json
+
+        - name: Apply route json configuration
+          shell: docker exec swss sh -c "swssconfig /vnet.route.json"
+
+        - name: sleep for some time
+          pause: seconds=3
+
+    - name: Remove VNET interfaces
+      command: docker exec -i database redis-cli -n 4 del "VLAN_INTERFACE|{{ item['ifname'] }}|{{ item['ip'] }}"
+      with_list: vlan_intf_list
+      become: true
+
```

Comment:
Can we also delete the Interface table without the IP prefix?

Reply:
Yes, should I also add it here?

Reply:
Yes.